.travis.yml

```diff
@@ -49,6 +50,10 @@ matrix:
     # which doesn't work on OS X.
     - os: osx
       env: CONFIG=golang
+    # HHVM currently uses apt-get to install hhvm and doesn't have a stable
+    # OS X build yet
+    - os: osx
+      env: CONFIG=hhvm
   include:
```

Comment:
I think you should add the test in include part instead of excluding the unused.